frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -585,3 +585,12 @@ export type SelfSubjectAccessReviewKind = {
     evaluationError?: string;
   };
 };
+
+export type DroppableEditYAMLProps = {
+  allowMultiple?: boolean;
+  obj: string;
```

Comment:
`obj` is an imprecise name since it's not an object, it's the YAML respresenting the obj. Maybe call this `content` or `yaml` or `resourceYAML`?

Reply:
Updating it to `content`.